tests/net/socket/socketpair/src/test_socketpair_happy_path.c

```diff
@@ -119,6 +119,7 @@ static void happy_path(
 			NULL, &len);
 
 		zassert_not_equal(res, -1, "recvfrom(2) failed: %d", errno);
```

Comment:
The original isn't quite redundant with the new check, but do you need both?  Why was `-1` special before?

Also: This can't be 210568, which is "Argument cannot be negative"

Reply:
I'll have another look again shortly

Reply:
The assert message is different. In the `!= -1` case, we print `errno`, and in the `>= -1` case, we're checking for valid range of the return. I could add comments to that effect if you think it's more clear.

Reply:
@pabigot - It is actually 201568. The issue is that the length argument to `strncmp()` cannot be negative because it's implicitly cast to `size_t`. So, the additional assertion is there to specifically prevent that corner case).
If you prefer, I can combine the two assertions to check for `>= 0' and also print `errno`.

Reply:
The 80-character limit only leaves so much room for messages unfortunately. In Linux 100 is the new limit ;-)